client/js/components/procedure/DpSubmitterList.vue

```diff
@@ -79,17 +88,18 @@ export default {
 
   data () {
     return {
-      headerFields: [
-        { field: 'name', label: Translator.trans('name'), initialMaxWidth: 250 },
-        { field: 'email', label: Translator.trans('email'), initialMaxWidth: 200 },
-        { field: 'postalCodeAndCity', label: Translator.trans('postalcode') + ' / ' + Translator.trans('city'), initialMaxWidth: 120 },
-        { field: 'organisationAndDepartment', label: Translator.trans('organisation') + ' / ' + Translator.trans('department'), initialMaxWidth: 200 },
-        { field: 'memo', label: Translator.trans('memo'), initialMinWidth: 100, initialMaxWidth: 200 },
-        { field: 'internId', label: Translator.trans('internId.shortened'), initialWidth: 80 },
-        { field: 'statement', label: Translator.trans('id'), tooltip: Translator.trans('id.statement.long'), initialWidth: 40 }
+      headerFieldsAvailable: [
+        { field: 'name', label: Translator.trans('name') },
+        { field: 'email', label: Translator.trans('email') },
+        { field: 'address', label: Translator.trans('city') },
```

Comment:
If this field contains the address, i'd propose to also use the label `address`.

```suggestion
        { field: 'address', label: Translator.trans('address') },
```

Reply:
![image](https://user-images.githubusercontent.com/91727189/214536791-a24daba5-d7ea-4fa8-ad25-206b15fe088e.png)

Since the address can contain a postalcode+city or a whole address I suggest to make this also visible in the table header like `Adresse / Ort`. I was expecting a city when seeing  `Ort` in the header field but I would also be expecting more than just a postal code when I see the header is `Adresse`

Reply:
thx! I’ve divided the address into two columns [here](https://github.com/demos-europe/demosplan-core/pull/672/commits/b71acab69b89cb56e6d1e80a829ff8c5a9a3ef25) (street and Postal Code / City). The first version was not correct. The street address should stay before the postal code in a separate column.